internal/service/dns_config/api_auth_zone_resource_test.go

```diff
@@ -507,13 +537,30 @@ func TestAccAuthZoneResource_TransferAcl(t *testing.T) {
 			},
 			// Update and Read
 			{
-				Config: testAccAuthZoneAclAny("tf-acc-test.com.", "cloud", "transfer_acl", "deny"),
+				Config: testAccAuthZoneAclAny(fqdn, "cloud", "transfer_acl", "deny"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAuthZoneExists(context.Background(), resourceName, &v),
 					resource.TestCheckResourceAttr(resourceName, "transfer_acl.0.access", "deny"),
 					resource.TestCheckResourceAttr(resourceName, "transfer_acl.0.element", "any"),
 				),
 			},
+			// Update and Read
+			{
+				Config: testAccAuthZoneAclAcl(fqdn, "cloud", "transfer_acl"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAuthZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "transfer_acl.0.element", "acl"),
+				),
+			},
+			//Update and Read
+			{
+				Config: testAccAuthZoneAclTsigKey(fqdn, "cloud", "transfer_acl", "deny"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAuthZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "transfer_acl.0.access", "deny"),
+					resource.TestCheckResourceAttr(resourceName, "transfer_acl.0.element", "tsig_key"),
```

Comment:
Please add check to verify Tsig key ID as well. 

Applies to all UTs

Reply:
Verification added